jenkins/prepare-yum-repos

```diff
@@ -27,15 +27,15 @@ STORAGE='http://cortx-storage.colo.seagate.com'
 
 cat <<EOF > /etc/yum.repos.d/motr_last_successful.repo
 [motr-dev]
-baseurl=$STORAGE/releases/eos/github/release/rhel-7.7.1908/last_successful/
+baseurl=$STORAGE/releases/cortx/github/main/centos-7.8.2003/last_successful/
 gpgcheck=0
 name=motr-dev
 enabled=1
 EOF
 
 cat <<EOF > /etc/yum.repos.d/lustre_release.repo
 [lustre]
-baseurl=$STORAGE/releases/cortx/uploads/rhel/rhel-7.7.1908/lustre/lustre-2.12.3/
+baseurl=$STORAGE/releases/cortx/third-party-deps/centos/centos-7.8.2003-2.0.0-latest/lustre/custom/tcp/
```

Comment:
@parikshit-dharmale you've got 11 commits in your PR but only 2 lines are changed. Can you please rebase your branch against the freshest `main` branch and squash your commits into one?